openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -56,17 +56,22 @@ This specification uses the terms "Holder", "Issuer", "Verifier", and "Verifiabl
 
 The following aspects are in scope of this interoperability profile:
 
-* Protocol for issuance of the Verifiable Credentials (can be both remote and in-person) (OID4VCI)
-* Protocol for online presentation of Verifiable Credentials (can be both remote and in-person) (OID4VP)
+* Profile of OpenID4VCI to issue IETF SD-JWT VCs, including
+  * Wallet Attestation
+* Profile of OpenID4VP to present IETF SD-JWT VCs
+* Profile of OpenID4VP over the W3C Digital Credentials API [@w3c.digital_credentials_api] to present 
+  * IETF SD-JWT VCs 
+  * ISO mdocs
 * Protocol for User Authentication by the Wallet at a Verifier (SIOP v2)
-* Wallet Attestation (during Credential issuance)
-* Credential Format (SD-JWT VC)
-* Status Management of the Credentials, including revocation
-* Cryptographic Holder Binding
-* Issuer key resolution
-* Issuer identification (as prerequisite for trust management)
+* Profile of IETF SD-JWT VC that includes the following aspects
+  * Status Management of the Credentials, including revocation
+  * Cryptographic Key Binding
```

Comment:
https://openid.github.io/OpenID4VP/openid-4-verifiable-presentations-wg-draft.html#section-2-4.26

Reply:
we can keep it as cryptographic key binding. i opened a issue here: https://github.com/openid/OpenID4VP/issues/369

Reply:
What do you mean with "our spec"? Definitely not in the SD-JWT,  as the changelog even says

> *  Use the term Key Binding rather than Holder Binding